news/changelog-1.6.md

```diff
@@ -29,6 +30,7 @@ All changes included in 1.6:
 - Fix `kbd` element styling on dark themes.
 - ([#10761](https://github.com/quarto-dev/quarto-cli/issues/10761)): Add support for `licence: CC0` to automatically link to Creative Commons licence [CC0 1.0](https://creativecommons.org/publicdomain/zero/1.0/).
 - ([#10817](https://github.com/quarto-dev/quarto-cli/issues/10817)): Ensure that user provided SCSS has precedent over quarto generated scss also for dark theme.
+- ([#11124](https://github.com/quarto-dev/quarto-cli/pull/11124)): Use stable order of GLightbox options
```

Comment:
I have added two entries because I think there are two aspects of this change: one refactoring which may have unsurfaced side effects and one bugfix / enhancement. please feel free to delete either of them.

Reply:
Ok. I'll remove the `html` format change.

Reply:
most users will probably not notice this anyway :sweat_smile: